eng/Version.Details.xml

```diff
@@ -42,5 +42,10 @@
       <Uri>https://github.com/dotnet/arcade</Uri>
       <Sha>2b9ccd49bbcca2156e229b00a140a8a018baac58</Sha>
     </Dependency>
+    <!-- Added manually - as System.Security.Cryptography.Pkcs is not darc subscribed -->
+    <Dependency Name="System.Security.Cryptography.Pkcs" Version="7.0.2">
```

Comment:
Should this be in "product" dependencies?

Reply:
Accidental leftover - removing